uwsift/view/tile_calculator.py

```diff
@@ -0,0 +1,624 @@
+import numpy as np
```

Comment:
Could you add the 2 lines about `#!/usr/bin/env` and encoding? Could you also add a docstring for the module?